### ISSUE TITLE
how to extract native m6A-modified and unmodified data of the 3106 m6A site  from VIRc and vir- 1 respectively?

### ISSUE THREAD

**Issue by hanyishui0612** [weir12/DENA#27]
Hi,thank you for your good job!I have a question.
How to extract the data in VIRc and vir1 respectively based on 3106 sites? Looking forward to your answer.Thank you !
![image](https://github.com/user-attachments/assets/084b01ff-a397-4c1f-98d8-082b805549d5)

**Comment by weir12** [comment]
Hi, to extract the data from VIRc and vir1 based on the 3106 sites, you can use the script found at [this GitHub link](https://github.com/weir12/DENA/blob/release/step2_feature_extraction/extract_signal_feature.py). This script should help with the extraction process. Let me know if you need any further assistance!

**Comment by hanyishui0612** [comment]
thank you for your reply!

**Comment by hanyishui0612** [comment]
I  am sorry to bother you .I want to solve a question for 3106 sites in Arabidopsis thaliana in the article.
![image](https://github.com/user-attachments/assets/2de3830a-d187-44ae-ab94-38c7c0e3d88a)
I don't know what the data in the last column represents.
Does it represent the position of the m6A site on the transcript?
Looking forward to hearing from you. Thank you.

**Comment by weir12** [comment]
Yes, that’s correct. The data in the last column represents the position of the m6A site on the transcript.

Best regards,
Ou
